codes/quantum/oscillators/fock_state/knill-laflamme-milburn-loqc.yml

```diff
@@ -0,0 +1,71 @@
+#######################################################
```

Comment:
Propose we merge this with code_id: dual_rail? This seems like a complete scheme on top that encoding, and would not warrant its own entry.

Reply:
@MikeVasmer

Reply:
Yes I agree

Reply:
@valbert4 agreed, I was wondering if this entry is redundant. Can dual_rail be a parent for this entry?